lib/twilio.rb

```diff
@@ -0,0 +1,21 @@
+require 'twilio-ruby'
+
+class Twilio_sms
+
+  def index
+  end
+
+  def send_text_message
+
+    account_sid= 'AC0b5e68dd8ffad76c7ca97787af3b4eca'
+    auth_token = '3837b88724c6df09e1dafba69b36266a'
+
+    @client = Twilio::REST::Client.new account_sid, auth_token
+
+    @client.account.sms.messages.create(
+      :from => '+441340202055',
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)